preupg/settings.py

```diff
@@ -173,13 +164,15 @@
 openssl_command = "openssl x509 -text -in {0} | grep -A1 1.3.6.1.4.1.2312.9.1"
 
 UPGRADE_PATH = ""
-KS_DIR = os.path.join(result_dir, 'kickstart')
-KS_TEMPLATE = 'default.ks'
-KS_TEMPLATE_POSTSCRIPT = 'finish.sh'
-KS_TEMPLATES = [KS_TEMPLATE, KS_TEMPLATE_POSTSCRIPT]
+KS_DIR = os.path.join(assesment_results_dir, 'kickstart')
+KS_FILENAME = 'migration.ks'
```

Comment:
This is mentioned in several Knowledge Base article. The change will have an affect to change the articles.
I would prefer to use preupgrade.ks filename as it was. Also QA tests will fail.